main/sdl2/APKBUILD

```diff
@@ -2,17 +2,18 @@
 # Maintainer: August Klein <amatcoder@gmail.com>
 pkgname=sdl2
 pkgver=2.0.7
-pkgrel=0
+pkgrel=1
 pkgdesc="A development library designed to provide low level access to audio, keyboard, mouse, joystick and graphics"
 url="http://www.libsdl.org"
 arch="all"
 license="zlib"
 depends=""
-makedepends="alsa-lib-dev libxcursor-dev libxrandr-dev libxrender-dev mesa-dev
+makedepends="alsa-lib-dev directfb-dev libxcursor-dev libxrandr-dev libxrender-dev mesa-dev
 	libxscrnsaver-dev libxi-dev libx11-dev libxext-dev libusb-dev libxkbcommon-dev"
 subpackages="$pkgname-dev"
 source="https://www.libsdl.org/release/SDL2-$pkgver.tar.gz"
 builddir="$srcdir/SDL2-$pkgver"
+options="!check"
```

Comment:
Please add a comment why you disabled tests. Does upstream have a test suite?

Reply:
Travis fails without it.. the original APKBUILD in Alpine didn't include any check function.. 

I can't find anything in SDL2 to run checks/tests after building, it doesn't appear like they implemented anything.